File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/SessionWeather/SessionWeatherOverlayDefinition.swift
import AppKit

enum SessionWeatherOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "session-weather",
        displayName: "Session / Weather",
        defaultSize: NSSize(width: 480, height: 520),
        fadeWhenLiveTelemetryUnavailable: true
    )
}

File: src/TmrOverlay.App/Overlays/SessionWeather/SessionWeatherOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.SessionWeather;

internal static class SessionWeatherOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "session-weather",
        DisplayName: "Session / Weather",
        DefaultWidth: 464,
        DefaultHeight: 496,
        FadeWhenLiveTelemetryUnavailable: true);
}
